repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/datamesh/DataDomainPreProcessor.java

```diff
@@ -290,10 +291,15 @@ private void moveChildrenToAnotherDomain(AtlasVertex childDomainVertex,
             String currentDomainQualifiedName = childDomainVertex.getProperty(QUALIFIED_NAME, String.class);
             String updatedDomainQualifiedName = parentDomainQualifiedName + getOwnQualifiedNameForChild(currentDomainQualifiedName);
 
+            String uniqueQualifiedNameAttribute = UNIQUE_ATTRIBUTE_SHADE_PROPERTY_PREFIX + QUALIFIED_NAME;
```

Comment:
Form this in Constant class as a static similar to QUALIFIED_NAME
No need to concatenate everytime